build-system/server.js

```diff
@@ -501,12 +501,14 @@ function replaceUrls(mode, file) {
     file = file.replace('https://cdn.ampproject.org/viewer/google/v5.js', 'https://cdn1.ampproject.org/viewer/google/v5.js');
     file = file.replace(/(https:\/\/cdn.ampproject.org\/.+?).js/g, '$1.max.js');
     file = file.replace('https://cdn.ampproject.org/v0.max.js', '/dist/amp.js');
+    file = file.replace('https://cdn.ampproject.org/inabox-v0.max.js', '/dist/amp-inabox.js');
     file = file.replace(/https:\/\/cdn.ampproject.org\/v0\//g, '/dist/v0/');
     file = file.replace('https://cdn1.ampproject.org/viewer/google/v5.js', 'https://cdn.ampproject.org/viewer/google/v5.js');
   }
   if (mode == 'min') {
     file = file.replace(/\.max\.js/g, '.js');
     file = file.replace('/dist/amp.js', '/dist/v0.js');
+    file = file.replace('/dist/amp-inabox.js', '/dist/inabox-v0.js');
```

Comment:
I think we need a different binary name. Something that follows `⚡` for ads symbol very closely? E.g. if it's `⚡4ads`, I'd call the file `amp4ads-v0.js` or just `ads-v0.js`.